### PR TITLE
feat-project-object-restructuring

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

Restructure project Object, more details on `CHANGES.md`

## Test Plan

Modified existing tests accordingly

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: app/config/auth.php [hunk 1]
```diff
@@ -5,35 +5,35 @@
...
-        'key' => 'usersAuthInvites',
+        'key' => 'invites',
         'icon' => '/images/users/invites.png',
         'docs' => 'https://appwrite.io/docs/client/teams?sdk=web#teamsCreateMembership',
         'enabled' => true,
     ],
     'jwt' => [
         'name' => 'JWT',
-        'key' => 'usersAuthJWT',
+        'key' => 'JWT',
```
Comment (TorstenDittmann) [inline app/config/auth.php:29]: I hate to see only capital letters, but I think this makes sense 😄

File: app/controllers/api/projects.php [hunk 2]
```diff
@@ -31,14 +31,6 @@
     }
 }, ['project'], 'projects');
 
-App::init(function ($project) {
-    /** @var Utopia\Database\Document $project */
-
-    if($project->getId() !== 'console') {
-        throw new Exception('Access to this API is forbidden.', 401);
```
Comment (TorstenDittmann) [inline app/controllers/api/projects.php:38]: Why do we remove this security measure that we just added?

File: docker-compose.yml [hunk 3]
```diff
@@ -39,7 +39,7 @@ services:
     build:
       context: .
       args:
-        - DEBUG=false
+        - DEBUG=true
```
Comment (TorstenDittmann) [inline docker-compose.yml:42]: leftover 😉

File: app/controllers/api/projects.php (hunk 2 above)
Comment (lohanidamodar) [reply app/controllers/api/projects.php:38]: It was there twice

File: docker-compose.yml (hunk 3 above)
Comment (lohanidamodar) [reply docker-compose.yml:42]: Ah yes

**Review by eldadfux — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**